moveit_planners/trajopt/trajopt/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(trajopt)
+
+add_compile_options(-std=c++11 -Wall -Wextra)
+
+find_package(catkin REQUIRED COMPONENTS
+  trajopt_sco
+  trajopt_utils
+  roscpp
+)
+
+find_package(Eigen3 REQUIRED)
+find_package(Boost COMPONENTS system python thread program_options REQUIRED)
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(JSONCPP jsoncpp)
+
+link_directories(
+    /usr/local/lib
+    /usr/local/lib64
+)
+
+set(TRAJOPT_SOURCE_FILES
```

Comment:
move this next to where its used in this file

Reply:
done